root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3>
+              <img
+                alt="MusicBrainz"
+                src="../../static/images/meb-logos/musicbrainz.svg"
+                width="180"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <p className="fs-4">
+              {l(`MusicBrainz is an open music encyclopedia that collects
+              music metadata and makes it available to the public.`)}
+            </p>
+            <ul className="list-unstyled">
+              <li>
+                <span className="fs-4">
+                  {l('Development IRC: ')}
+                </span>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-4">
+                  {l('Email: ')}
                 </span>
-              ),
-            })}
-          </>
-        ) : null}
+                <a
+                  className="fw-bold fs-4"
+                  href="mailto:support@metabrainz.org"
+                >
+                  {l('support@metabrainz.org')}
+                </a>
+              </li>
+            </ul>
+            {serverLanguages && serverLanguages.length > 1 ? (
+              <LanguageMenu
+                $c={$c}
+                currentBCP47Language={
+                  $c.stash.current_language.replace('_', '-')
+                }
+                serverLanguages={serverLanguages}
+              />
+            ) : null}
+          </div>
+          <br />
+          <div className="col-sm-12 col-md-3">
+            <h3 className="fs-2 fw-bold color-black">
+              {l('Useful Links')}
+            </h3>
+            <ul className="list-unstyled">
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
```

Comment:
This should either use CSS's `list-style-image` or `list-style-type` (probably the latter, since the image is just of a normal character that we can use).

Reply:
Fixed